std/mmfile.d

```diff
@@ -134,19 +134,25 @@ class MmFile
             char c = 0;
             core.sys.posix.unistd.write(fd, &c, 1);
         }
-        else if (prot & PROT_READ && size == 0)
-            size = statbuf.st_size;
+        else if (prot & PROT_READ && size == 0) {
+            if(statbuf.st_size > 0) size = statbuf.st_size;
+            else return;
```

Comment:
You should avoid the if/else/return pattern. See

https://www.yegor256.com/2015/01/21/if-then-throw-else.html?2015-03